frontend/src/lib/user.ts

```diff
@@ -179,6 +179,10 @@ function jwtToUser(user: JwtTokenUser): LexAuthUser {
   const { sub: id, name, email, user: username, proj: projectsString, role: jwtRole } = user;
   const role = Object.values(UserRole).find(r => r.toLowerCase() === jwtRole) ?? UserRole.User;
 
+  if (user.orgs) {
+    user.orgs = user.orgs.map(o => ({ orgId: o.OrgId ?? o.orgId, role: (o.Role ?? o.role).toUpperCase() as OrgRole } ));
```

Comment:
This produces type errors because the type of `user.orgs` will never have OrgId and orgId properties, etc., but I think I want to keep this so that we don't need to change the frontend code again if we change the property serialization format in the backend. So I plan to just suppress type errors on this line. Let me know if you want a different solution to the Typescript type errors.